libsql/internal/hrana/batch.go

```diff
@@ -1,7 +1,13 @@
 package hrana
 
+import (
+	"encoding/json"
+	"fmt"
+)
+
 type Batch struct {
-	Steps []BatchStep `json:"steps"`
+	Steps            []BatchStep `json:"steps"`
+	ReplicationIndex *uint64     `json:"replication_index"`
```

Comment:
Do we need omitempty here?

Reply:
Here, we need. I have added in the subsequent comments.